test-packages/test-services-openapi/no-schema-service/default-api.d.ts

```diff
@@ -9,6 +9,7 @@ import { OpenApiRequestBuilder } from '@sap-cloud-sdk/openapi';
  * This API is part of the 'no-schema-service' service.
  */
 export declare const DefaultApi: {
+  _defaultBasePath: string;
```

Comment:
[q] Why is it string here and undefined else where?

Reply:
Good catch, I have no idea, why, let me check.

Reply:
I think this is expected, because in this example, we explicitly set `basePath` to a custom value.
So, the corresponding `.ts` file entry is `_defaultBasePath: '/base/path/to/service'`.

In all the other tests, the `basePath` value is `undefined` and hence the declaration files also have the same value.
Is this an issue?

Reply:
Oops, I mistook this for a .ts file. Didn't realize its a declaration file. Then please ignore my comment,